Source/WebCore/dom/Range.h

```diff
@@ -148,9 +148,9 @@ class Range final : public AbstractRange, public CanMakeSingleThreadWeakPtr<Rang
     Ref<Document> m_ownerDocument;
     RangeBoundaryPoint m_start;
     RangeBoundaryPoint m_end;
-    bool m_isAssociatedWithSelection { false };
-    bool m_didChangeForHighlight { false };
-    bool m_isAssociatedWithHighlight { false };
+    bool m_isAssociatedWithSelection : 1 { false };
```

Comment:
I don't think this will really help on its own because of the padding needed to make the whole thing word aligned at least in 64-bit architectures. But if we re-ordered members of RangeBoundaryPoint as well so that m_offset is at the end, it might result in better packing. You can use `Tools/Scripts/dump-class-layout` to dump the structure packing although this tool seems to segfault in macOS Sequoia...

Reply:
I agree with Ryosuke that this likely doesn't help anything. You can easily verify this by printing the size of the class before & after (https://stackoverflow.com/a/2008577)